changed to a new project in firebase

ItayNaumann opened the pull request.

ItayNaumann commented:
could be merged later by rebasing with little to no conflicts so look at the other pr first

giladwo commented:
> could be merged later by rebasing with little to no conflicts so look at the other pr first

I don't see why would it be easier to look at #380 with plenty other unrelated changes cluttering the diff.


Anyways, there are many API keys here, which should almost definitely not be public.
Simply not merging these changes isn't good enough - they're all still available right here.

https://github.com/orgs/Team1690/teams/scouting Search firebase docs and find which of these should be kept off version control and need to be changed now that they've leaked.

ItayNaumann commented:
the permissions we give here are to read the console and send info to the console, I deleted the project cause there is no need for changing the storage cause we don't need to save last year's photos. In the case of hosting, we have github to fall back for past versions so I think I wont change the firebase projects. these api keys have no use cause i deleted the project and we are still on the wanted projects so no need to change.
